src/collections/_documentation/cli/dif/elf.md

```diff
@@ -25,6 +25,50 @@ information in separate `*-dev` packages and put it in locations like
 libraries, make sure to also upload their symbols in addition to your app’s
 symbols.
 
+## Build Identifiers
+
+Sentry uses the GNU build identifier to associate uploaded debug information
+files with crashes. All recent compilers and linkers support the emission of
+build IDs, but sometimes they might require additional configuration. `gcc` does
+this by default, for `clang` use one of the following flags:
+
+* `--build-id=uuid` for a fast but non-reproducible random identifier.
+* `--build-id=sha1` for a slower but reproducible identifier generated by
+  hashing the first page of the code section.
+
+Note that the identifier needs to be present and identical in the binary as well
+as stripped debug information files. If the ID is missing for some reason,
+upload the files before stripping so that `sentry-cli` can compute a stable
+identifier from the unstripped file. This can be verified with our `difutil`
+command:
+
+```bash
+ $ sentry-cli difutil check path/to/file
+
+Debug Info File Check
+  Type: elf library
+  Contained debug identifiers:
+    > 924e148f-3bb7-06a0-74c1-36f42f08b40e (x86_64)
+  ...
+```
+
+## Debug Info Compression
+
+ELF supports the compression of debug information which can significantly
+reduce the time required to upload debug information files to Sentry and thus
+improve build times. `gcc` and recent versions of `clang` support this by
```

Comment:
Please specify the versions of clang that supports compression of debug information.